freeipa/src/main/java/com/sequenceiq/freeipa/converter/credential/CredentialV4RequestToCredentialConverter.java

```diff
@@ -16,7 +17,7 @@
     @Override
```

Comment:
This probably doesn't convert CredentialV4Requests anymore after all the refactoring. Rename to CredentialRequestToCredentialConverter.

I also noticed that some other converters no longer implement Converter<?,?>. e.g., InstanceTemplateRequestToTemplateConverter and InstanceGroupRequestToInstanceGroupConverter. Should they all implement Converter or not?

Reply:
Converter supports only one parameter, and I want to use more. As I don't think we want to use ConversionService to get converters, implementing the interface is not necessary

Reply:
thanks for the info!